app/models/miq_expression.rb

```diff
@@ -1016,8 +1016,8 @@ def self.exclude_col_by_preprocess_options?(col, options)
     Metric::Rollup.excluded_col_for_expression?(col.to_sym)
   end
 
-  def self.evaluate(expression, obj, tz = nil)
-    ruby_exp = expression.kind_of?(Hash) ? new(expression).to_ruby(tz) : expression.to_ruby(tz)
```

Comment:
I note there's a very subtle behavior change where we (sadly) have different behavior when going [through the instance method first](https://github.com/manageiq/manageiq/blob/7af0731a671a5426697eebd612046508755f7760/app/models/miq_expression.rb#L436-L442) This code looks deliberate, but I'm pretty unfamiliar with the ins and outs of MiqExpression. Are you sure this line is removable or something in `to_ruby` doesn't need to be edited as well?